### ISSUE TITLE
displaying a dataframe with no column data

### ISSUE THREAD

**Issue by jgunstone** [bloomberg/ipydatagrid#392]
**Describe the bug**

when passed a dataframe with index but no column data ipydatagrid returns an error

**To Reproduce**

```py
import pandas as pd
from ipydatagrid import DataGrid

df = pd.DataFrame(columns=["a", "b"])
DataGrid(df.T)
```

**Expected behavior**

expect equivalent to when a dataframe with no row data is passed, 
i.e.
```py
import pandas as pd
from ipydatagrid import DataGrid

df = pd.DataFrame(columns=["a", "b"])
DataGrid(df)
```
![image](https://user-images.githubusercontent.com/21370980/214888497-f7e1f4e3-7d99-47b8-bbf1-a6110b41472e.png)

**Screenshots**
![image](https://user-images.githubusercontent.com/21370980/214888184-a71343e9-369f-4a8e-a684-b0c361938e3e.png)

**Environment (please complete the following information):**
 - Operating System and Version: Ubuntu 20:04
 - Browser [e.g. Firefox, Safari] (if applicable): Chrome
 - ipdatagrid version: ipydatagrid           1.1.14             pyhd8ed1ab_0    conda-forge

**Comment by martinRenou** [comment]
Fixed by #480

**Comment by jgunstone** [comment]
thanks!
